lib/full_text_search/custom_field_mapper.rb

```diff
@@ -0,0 +1,36 @@
+module FullTextSearch
+  class CustomFieldMapper < Mapper
+    class << self
+      def redmine_mapper_class
+        RedmineCustomFieldMapper
+      end
+
+      def fts_mapper_class
+        FtsCustomFieldMapper
+      end
+    end
+  end
+  resolver.register(CustomField, CustomFieldMapper)
+
+  class RedmineCustomFieldMapper < RedmineMapper
+    class << self
+      def with_project(redmine_class)
+      end
+
+      def not_mapped(redmine_class, options)
```

Comment:
Question: What is `not_mapped`? When we need this method? Do we need do some thing in this method?

Reply:
`not_mapped` is for collecting Redmine records that are not mapped (corresponding `FullTextSearch::Target` doesn't exist). It's used to synchronize Redmine records and `FullTextSearch::Target.